en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -1,96 +1,98 @@
-        1. Click **+ Create** to start creating a new service.
-        2. Enter a name for the service and click **Create**.
-           
-           ![Create a new service](../assets/img/observability/service-creation.png){.cInlineImage-full}
+    You are directed to the sample service low-code view where you can see the following:
 
-3. From the left-hand menu, click **Observe**.
-4. On the right-hand bottom, click the **Sample Service** icon.
-   
-    ![Sample service popup](../assets/img/observability/sample-service-pop-up.png){.cInlineImage-full}
-   
-5. Click **Try Sample Service**.
+    - Low-code diagram
+    - Throughput graph   
+    - Latency graph
+    - Logs
+    - Diagnostic data
 
-    ![Try sample service](../assets/img/observability/try-sample-service.png){.cInlineImage-half}
+Here, you can detect anomalies and perform root cause analysis on the sample service.
     
 ## Detect anomalies and perform root cause analysis
-You can detect the anomalies of the usage of a service by observing the **Throughput & Latency** graphs. The throughput graph shows the successful and erroneous requests that occurred during a period. The latency graph shows the latencies of each request. You can observe these graphs at a lower granularity by choosing a shorter time range, which gives you a better understanding of the incidents.
 
-### Step 1: Detect the anomalies
+To detect anomalies of the service, you must trace its executions from the code statement level to the machine it runs in. You can analyze observability data and logs to troubleshoot and identify the root cause for the anomalies you detect. 
 
-#### Analyze the throughput graph
-The throughput graph shows the throughput of requests per hour for a selected timestamp. 
 
-With the sample data, you can observe spikes in the throughput graph.  You can find the root cause for this by learning how to analyze the throughput graph.
+### Analyze the throughput graph
 
-Follow the steps below. 
+The throughput graph depicts the total throughput of the service (which includes both successful and erroneous requests) during a selected time interval. You can analyze the spikes in the throughput graph to identify the root cause for requests that result in an error.
 
-1. Hover over the throughput graph where you see spikes. You can observe the number of errors that occurred at the time. 
+The following image is an example of a throughput graph:
+
+![Analyze the throughput graph](../assets/img/observability/throughput-graph-analysis.png){.cInlineImage-full}
+
+The actions you can perform are as follows:
+
+- **View the number of successful requests and errors**
+
+    To do this, hold the pointer over the area of the graph that shows the required time interval. The Observability view displays the number of successes over the graph as shown in the above image (in this example it displays 296 successes and 2512.43 errors). The success rate of each connector is shown in the low-code diagram area.
```

Comment:
Doesn't seem correct to say 
"The Observability view displays the number of successes over the graph as shown in the above image". 
and
"The success rate of each connector is shown in the low-code diagram area."

Reply:
Rephrased it as follows:

"The throughput graph displays the number of successes over the graph as shown in the above image (in this example it displays 296 successes and 2512.43 errors). The low-code diagram displays the success rate for each connector."